### DISCUSSION TITLE
How to change the preview of tab hover?

### DISCUSSION THREAD

**Discussion by RashiqAzhan** [onemen/TabMixPlus#347] [Q&A] [answered]
I get a preview of the content of the tab when I hover over it.
Like so.
![image](https://github.com/user-attachments/assets/561c3183-3f0c-4b08-8676-359dc284522c)

Is there a way to customize it or turn it off? The stock browser showed the full title.

Long titles get truncated for me at the moment.

Is it possible to scroll the entire tab while still hovering over the tab?

Thank you for your project. Much appreciated.

**Comment by 117649** [comment, marked as answer]
https://support.mozilla.org/bm/questions/1459859
